chef-utils/lib/chef-utils/dsl/architecture.rb

```diff
@@ -103,6 +103,17 @@ def powerpc?(node = __getnode)
         %w{powerpc}.include?(node["kernel"]["machine"])
       end
 
+      # Determine if the current architecture is arm
+      #
+      # @since 15.10
+      #
+      # @return [Boolean]
+      #
+      def arm?(node = __getnode)
+        %w{armhf aarch64 arm64 arch64}
+        .include?(node["kernel"]["machine"])
```

Comment:
Super nitpick but using two lines like this is kind of a weird way to write this statement.

Reply:
True. Fixed